debuggers/gdb/shim.js

```diff
@@ -232,9 +232,9 @@ function Executable() {
     this.spawn = function(callback) {
         var args = ["--once", ":"+GDB_PORT, BIN].concat(ARGS);
         this.proc = spawn("gdbserver", args, {
-            detached: false,
+            detached: true,
```

Comment:
why is detached flag needed? It seems like it has effect only when stdio is 'ignore'